resources/local/podman/vm.go

```diff
@@ -40,48 +36,45 @@ func NewInstance(e resourceslocal.Environment, args VMArgs, opts ...pulumi.Resou
 	dataPath := path.Join(homeDir, ".localpodman")
 	// TODO clean up the folder on stack destroy
 	// Requires a Runner refactor to reuse crossplatform commands
-	err = os.MkdirAll(dataPath, 0700)
+	dataDir, err := fileManager.CreateDirectory(dataPath, false)
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review. Do look for something to sincerely praise. -->
**👏 praise**
Yay!